manifests/mg.pp

```diff
@@ -4,9 +4,11 @@
   $homedir = '/root',
   $default = true,
 ) {
+  contain epel
```

Comment:
yeah, I was thinking because it was a class it would be better to use contain, but I think you're right.